bitnami/prometheus-operator/templates/prometheus/prometheus.yaml

```
@@ -137,6 +137,11 @@ spec:
     {{- if .Values.prometheus.podAffinity }}
     podAffinity: {{- include "prometheus-operator.tplValue" (dict "value" .Values.prometheus.podAffinity "context" $) | nindent 6 }}
     {{- end }}
+    {{- if or .Values.prometheus.additionalScrapeConfigsExternal }}
```

Had noticed this one in the last review. Completely missed adding a note about it :man_facepalming: 

```suggestion
    {{- if .Values.prometheus.additionalScrapeConfigsExternal }}
```